README.md

```diff
@@ -2,17 +2,29 @@
 
 RL Swarm is a peer-to-peer system for reinforcement learning. It allows you to train models collaboratively with others in the swarm, leveraging their collective intelligence. It is open source and permissionless, meaning you can run it on a consumer laptop at home or on a powerful GPU in the cloud. You can also connect your model to the Gensyn Testnet to receive an on-chain identity that tracks your progress over time.
 
-Currently, we are running the [reasoning-gym](https://github.com/open-thought/reasoning-gym/tree/main) swarm on the Testnet. This swarm is designed to train models to solve a diverse set of reasoning tasks using the reasoning-gym dataset. The current list of default models includes:
+> **Update -> CodeZero Environment**  
+> RL Swarm now powers **CodeZero**, the newest active environment on the Gensyn Testnet.  
+> CodeZero extends RL Swarm beyond reasoning and math into **collaborative coding tasks**, where models take on distinct roles, *Solvers*, *Proposers*, and *Evaluators*, to learn from one another across a decentralized network.
 
-Models:
-   - Gensyn/Qwen2.5-0.5B-Instruct
-   - Qwen/Qwen3-0.6B
-   - nvidia/AceInstruct-1.5B
-   - dnotitia/Smoothie-Qwen3-1.7B
-   - Gensyn/Qwen2.5-1.5B-Instruct
+Currently, we are running **CodeZero** on the Gensyn Testnet.  
+
+CodeZero replaces the previous *reasoning-gym* environment, introducing a new task domain focused on programming challenges and model-based evaluation.  
+
+If you previously ran RL Swarm during the reasoning-gym phase, your node will automatically transition to CodeZero after updating using the `git pull` command. 
```

Comment:
they should also remove their old venv if using the `run_rl_swarm.sh` script and create a new one.  

```bash
git pull
rm -rf .venv
python -m venv .venv
source .venv/bin/activate
```